devices/lepton/image14bit/color.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package image14bit
+
+import (
+	"image/color"
+	"strconv"
+)
+
+// Color14 implements a 14-bit color.
```

Comment:
I'd prefer Intensity14, Color is misleading IMHO. Comments could be:

// Intensity14 is a 14 bits grayscale implementation of color.Color
//
// Valid range is between 0 and 16383.

Reply:
Done. Agreed that Intensity14 is a better name.